checker/jdk/nullness/src/java/lang/Class.java

```diff
@@ -39,7 +39,7 @@ protected Class() {}
   public java.lang.reflect.Method[] getMethods() throws SecurityException { throw new RuntimeException("skeleton method"); }
   public java.lang.reflect.Constructor<?>[] getConstructors() throws SecurityException { throw new RuntimeException("skeleton method"); }
   public java.lang.reflect.Field getField(String a1) throws NoSuchFieldException, SecurityException { throw new RuntimeException("skeleton method"); }
-  public java.lang.reflect.Method getMethod(String a1, Class<?> @Nullable ... a2) throws NoSuchMethodException, SecurityException { throw new RuntimeException("skeleton method"); }
+  public java.lang.reflect.Method getMethod(String a1, Class<?> ... a2) throws NoSuchMethodException, SecurityException { throw new RuntimeException("skeleton method"); }
```

Comment:
The method might require parameters, so to be consistent with the other reflective methods, the parameter is changed to be non-null.